editProfilePage.php

```diff
@@ -0,0 +1,65 @@
+<?php  
+    include('config/init.php');
+    echologgedInHeader("edit profile");
```

Comment:
You probably want to check to make sure they're logged in here. Don't let this stop getting this version deployed, but you should probably add this on a different branch.

This applies to all the other logged in pages as well.